README.md

```diff
@@ -7,21 +7,12 @@ animal_list = ["hare", "rabbit", "tortoise", "goat", "hound", "duck", "ducking",
 
 def isAnimal(value):
     if value in animal_list:
```

Comment:
Instead of if ...: return True, else return False, you can simply write
`
return value in animal_list;
`